qemu/tests/qemu_guest_agent.py

```diff
@@ -1163,6 +1163,9 @@ def get_new_disk(disks_before_plug, disks_after_plug):
                 disk_index = utils_misc.wait_for(
                     lambda: utils_disk.get_windows_disks_index(session, image_size_stg0), 120)
                 if disk_index:
+                    logging.info("Clear readonly for disk and online it in windows guest.")
+                    if not utils_disk.update_windows_disk_attributes(session, disk_index):
+                        test.error("Failed to update windows disk attributes.")
```

Comment:
For ling 1168("test.error("Failed to update windows disk attributes.")"), It is one step of this case ?  suggest to call the method test.fail if it is a step indeed.
Thanks.

Reply:
No,it's not a step.